modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -87,14 +89,12 @@ $ ./openshift-install create manifests --dir=<installation_directory> <1>
 .Example output
 [source,terminal]
 ----
-INFO Consuming Install Config from target directory
-WARNING Making control-plane schedulable by setting MastersSchedulable to true for Scheduler cluster settings
```

Comment:
Due to the removal of the step discussed [here](https://github.com/openshift/openshift-docs/pull/28079/files#r543245591), the warning does not appear because the nodes are no longer set to be schedulable when the manifests are created.